dotCMS/src/main/java/com/dotcms/ema/proxy/MockServletOutputStream.java

```diff
@@ -0,0 +1,60 @@
+package com.dotcms.ema.proxy;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.WriteListener;
+
+/**
+ * A mock Outputstream that will write the output of a servlet to a outputstream
+ * 
+ * @author will
+ *
+ */
+class MockServletOutputStream extends ServletOutputStream {
+    final OutputStream os;
+    
+    public MockServletOutputStream(ByteArrayOutputStream captureMe) {
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5922984)